test-result-summary-client/src/Build/ReleaseSummary.jsx

```diff
@@ -23,8 +23,15 @@ const ReleaseSummary = () => {
             const nl = `\n`;
             if (build && build[0]) {
                 const { buildName, buildUrl, timestamp, startBy } = build[0];
+                let buildsResultOutput = '';
+                if (childBuildsResult === 'PROGRESSING') {
+                    buildsResultOutput =
+                        `**Warning:** TRSS is currently in the process of retrieving and analyzing the build results. Presently, the release summary report only displays partial results. ` +
+                        `For the comprehensive release summary report, please verify its availability upon the completion of the TRSS processing.${nl}${nl}`;
```

Comment:
This is quite a lot of words for saying that "the report is not yet complete".  I wonder if we should shorten this message.

Reply:
The message is updated